bundles/org.openhab.io.openhabcloud/README.md

```diff
@@ -116,11 +116,9 @@ The parameters for these actions have the following meaning:
 
 The `sendNotification` and `sendBroadcastNotification` actions additionally support setting a title, reference id, media attachments and actions.
 
-- The tag is used for tagging messages for grouping when displaying in the app and to hide/remove groups of messages from a user's device.
-Tag was previously referred to as "severity".   
+- The tag is used for tagging messages for grouping when displaying in the app and to hide/remove groups of messages from a user's device. Tag was previously referred to as "severity".
 - The title is displayed as the notification title on the device and defaults to "openHAB" for the Android and iOS apps.
-- The reference id is a user supplied identifier that when set will replace messages with the same id on the user's device (so only the last version exists). 
-The reference id is a user supplied id that can be used to update or remove notifications with this same id.
+- The reference id is a user supplied identifier that when set will replace messages with the same id on the user's device (so only the last version exists).  It can be used to update or remove notifications.
```

Comment:
```suggestion
- The reference id is a user supplied identifier that when set will replace messages with the same id on the user's device (so only the last version exists). It can be used to update or remove notifications.
```